Rocket.Chat/Controllers/Chat/ChatControllerAutocomplete.swift

```diff
@@ -40,6 +40,12 @@ extension ChatViewController {
                 searchResult[channel.name] = channel.type == .channel ? UIImage(named: "Hashtag") : UIImage(named: "Lock")
             }
 
+        } else if prefix == "/" && word.count > 0 {
+            let commands = realm.objects(Command.self).filter("command BEGINSWITH[c] %@", word)
+
+            for command in commands {
+                searchResult[command.command] = command
+            }
         }
 
         let show = (searchResult.count > 0)
```

Comment:
I noticed that the results are only appearing if the text isn't empty. Can we should all the commands/usernames/chat right after user types `#` or `@` or `/`?